src/AutosizeInput.js

```diff
@@ -58,12 +59,18 @@ const AutosizeInput = React.createClass({
 			placeholderNode.style.letterSpacing = inputStyle.letterSpacing;
 		}
 	},
+	onInputChange (event) {
```

Comment:
Hey, I noticed you aren't calling `this.props.onChange` that has been passed on props. How would a parent component have access to the value when it changes. I think you need to add the following:

```
this.props.onChange ? this.props.onChange(value);
```